docs/cli/edgedb_instance/edgedb_instance_create.rst

```diff
@@ -20,6 +20,13 @@ instance and creating a corresponding credentials file in
 ``<edgedb_config_dir>/credentials``. Run ``edgedb info`` to see the path to
 ``<edgedb_config_dir>`` on your machine.
 
+.. note::
+
+    The ``edgedb instance create`` command is not intended for use with
+    self-hosted production instances. You can follow one of our
```

Comment:
This should also not be used for self-hosted staging or test instances. Maybe just drop `production`? Same for all the self-hosted references.

Reply:
Done.